PrivateCollections/Ethiopia/MamherDegguAddet/MDA003.xml

```diff
@@ -30,42 +31,260 @@
+                           <textLang mainLang="gez"/>
+                       </msItem>
+                       <msItem xml:id="ms_i1.6">
+                           <locus from="69vb" to="82ra"/> 
+                           <title type="complete" ref="LIT6843Commentary#CommentaryDeuteronomy"/>
+                           <textLang mainLang="gez"/>
+                       </msItem>
+                   </msItem>
+                       <msItem xml:id="ms_i2">
+                           <locus from="84ra" to="86rc"/> 
+                           <title type="complete" ref="LIT6849Menhar"/>
+                           <textLang mainLang="gez"/>
+                       </msItem>
+                       <msItem xml:id="ms_i3">
+                           <locus from="92ra" to="102va"/> 
+                           <title type="complete" ref="LIT3850MashafaM"/>
+                           <textLang mainLang="gez"/>
+                       </msItem>
+                       <msItem xml:id="ms_i4">
+                           <locus from="102vb" to="118rc"/> 
+                           <title type="complete" ref="LIT4302Maccab1"/>
+                           <textLang mainLang="gez"/>
+                       </msItem>
+                       <msItem xml:id="ms_i5">
+                           <locus from="118va" to="131vb"/> 
+                           <title type="complete" ref="LIT4303Maccab2"/>
+                           <textLang mainLang="gez"/>
+                       </msItem>
+                       <msItem xml:id="ms_i6">
+                           <locus from="135ra" to="169r"/> 
+                           <title type="complete" ref="LIT6676CommDaniel"/>
+                           <textLang mainLang="gez"/>
+                       </msItem>
                    </msContents>
                     <physDesc>
                         <objectDesc form="Codex">
                           <supportDesc>
                              <support> <material key="parchment"></material></support>
                              <extent>
                                  <measure unit="leaf">178</measure>
+                                 <measure unit="page" type="blank">33</measure>
                               <dimensions type="outer">
                                   <height unit="mm">350</height>
                              <width unit="mm">280</width><depth>90</depth>
                               </dimensions>
-                             </extent><condition key="good"></condition>
+                             </extent>
+                              <foliation> The manuscript is not foliated. The folio numbers used in this description were added digitally 
+                                  to the manuscript's images. </foliation>
+                              <collation>
+                                  <signatures>Decorated quire marks from <ref target="#q3"/>to <ref target="#q11"/> 
```

Comment:
I think it is "from ... to ... and **on** ...". Without "on" it means rather "from ... to ... and to ..."